File: Java OOP/Homework05/models/Programmer.java
package models;

public class Programmer extends Person {
    
    private int salary;
    private String qualification;

    public Programmer(int age, String surname, String name, String patronymic, String telephone, int salary,
            String qualification) {
        super(age, surname, name, patronymic, telephone);
        this.salary = salary;
        this.qualification = qualification;
    }

    public int getSalary() {
        return this.salary;
    }
    
    public String getQualification() {
        return this.qualification;
    }

    public void setSalary(int salary) {
        this.salary = salary;
    }
    
    public void setQualification(String qualification) {
        this.qualification = qualification;
    }   
}

File: Java OOP/Homework07/models/Programmer.py
from models.Person import Person


class Programmer(Person):
    def __init__(self, age: int, surname: str, name: str, patronymic: str, telephone: str, salary: int, qualification: str):
        super().__init__(age, surname, name, patronymic, telephone)
        self.salary = salary
        self.qualification = qualification

    def get_salary(self) -> int:
        return self.salary
    
    def get_qualification(self) -> str:
        return self.qualification

    def set_salary(self, salary: int):
        self.salary = salary
    
    def set_qualification(self, qualification: str):
        self.qualification = qualification
